src/components/Searchbar.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
```

Comment:
Here, you missed a few steps. You needed a `handleSubmit` function to make the API call, and use it on the input form. Also, we're going to use the `setBeer` props here.

Here's what your code should look like for it to work:
<img width="1290" alt="Screenshot 2023-03-06 at 12 58 48" src="https://user-images.githubusercontent.com/107497031/223117517-906f16f0-fe90-4641-a3ef-037b75231cf0.png">